multicompanyaccounting.civix.php

```diff
@@ -449,5 +449,11 @@ function _multicompanyaccounting_civix_civicrm_alterSettingsFolders(&$metaDataFo
  * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_entityTypes
  */
 function _multicompanyaccounting_civix_civicrm_entityTypes(&$entityTypes) {
-  $entityTypes = array_merge($entityTypes, []);
+  $entityTypes = array_merge($entityTypes, [
+    'CRM_Multicompanyaccounting_DAO_Company' => [
+      'name' => 'Company',
+      'class' => 'CRM_Multicompanyaccounting_DAO_Company',
+      'table' => 'civicrm_company',
```

Comment:
The table name in other places is `multicompanyaccounting_company`

Reply:
fixed